doc/reference/bluetooth/shell/audio.rst

```diff
@@ -0,0 +1,479 @@
+   "[start]","enable","enabling","streaming"
+   "disable","enable", "enabling/streaming","disabling"
+   "[stop]","disable","disabling","qos-configure/idle"
+   "release","config","any","releasing/codec-configure/idle"
+   "list","none","any","none"
+   "select","none","any","none"
+   "connect","discover","idle/codec-configured/qos-configured","codec-configured"
+   "send","enable","streaming","none"
+
+Example Central
+***************
+
+Connect and establish a stream:
+
+.. code-block:: console
+
+   uart:~$ bt init
+   uart:~$ audio init
+   uart:~$ bt connect <address>
+   uart:~$ gatt exchange-mtu
+   uart:~$ audio discover sink
+   uart:~$ audio config sink 0
+   uart:~$ audio qos
+   uart:~$ audio enable
+
+Or using connect command:
+
+.. code-block:: console
+
+   uart:~$ bt init
+   uart:~$ audio init
+   uart:~$ bt connect <address>
+   uart:~$ gatt exchange-mtu
+   uart:~$ audio discover sink
+   uart:~$ audio connect sink 0
+
+Disconnect and release:
+
+.. code-block:: console
+
+   uart:~$ audio disable
+   uart:~$ audio release
+
+Example Peripheral
+******************
+
+Listen:
+
+.. code-block:: console
+
+   uart:~$ bt init
+   uart:~$ audio init
+   uart:~$ bt advertise on
+
+Server initiated disable and release:
+
+.. code-block:: console
+
+   uart:~$ audio disable
+   uart:~$ audio release
+
+Example Broadcast Source
+************************
+
+Create and establish a broadcast source stream:
+
+.. code-block:: console
+
+   uart:~$ audio init
+   uart:~$ audio create_broadcast
+   uart:~$ audio start_broadcast
+
+Stop and release a broadcast source stream:
+
+.. code-block:: console
+
+   uart:~$ audio stop_broadcast
+   uart:~$ audio delete_broadcast
+
+
+Example Broadcast Sink
+************************
+
+Scan for and establish a broadcast sink stream:
+
+.. code-block:: console
+
+   uart:~$ audio init
+   uart:~$ audio broadcast_scan on
+   Found broadcaster with ID 0xB91CD4
+   uart:~$ audio accept_broadcast 0xB91CD4
+   PA syncing to broadcaster
+   Broadcast scan was terminated: 0
+   PA synced to broadcaster with ID 0xB91CD4 as sink 0x2000d09c
+   Sink 0x2000d09c is set as default
+   Sink 0x2000d09c is ready to sync without encryption
+   Received BASE from sink 0x2000d09c:
+   Subgroup[0]:
+   codec 0x06 cid 0x0000 vid 0x0000 count 4
+   data #0: type 0x01 len 1
+   data #1: type 0x02 len 1
+   data #2: type 0x03 len 4
+   00000000: 00 00 00                                         |...              |
+   data #3: type 0x04 len 2
+   00000000: 28                                               |(                |
+   meta #0: type 0x02 len 2
+   BIS[0] index 0x01
+   [0]: 0x01
+   Possible indexes: 0x01
+   audio sync_broadcast 0x01
+
+Stop and release a broadcast sink stream:
+
+.. code-block:: console
+
+   uart:~$ audio stop_broadcast_sink
+   uart:~$ audio term_broadcast_sink
+
+Init
+****
+
+This command register local PAC records which are necessary to be able to
```

Comment:
Even though the commands are listed above in the file, I think it makes sense to mention them (again) here, instead of saying "this command" without actually specifying which command "this" is.

Proposal here, for init:
The "init" command" registers local PAC records [...]  (note also the "s" in registers)

Proposal for discover (below):
Once connected, the "discover" command is used to discover [...]

Ditto for remaining commands